### DISCUSSION TITLE
How to get the files used in the example, for example **.mhd

### DISCUSSION THREAD

**Discussion by 888yyh** [pyvista/pyvista#4949] [Q&A] [answered]
How to get the files used in the example, for example **.mhd
i want to implement the examples, how can i download the files used in the tutorial?
thanks

**Comment by tkoyama010** [comment, marked as answer]
You can get the path of the file downloaded by the load function by setting the load argument of the download function to `False`.
```python
>>> from pyvista import examples
>>> dataset = examples.download_head()
>>> filename = examples.download_head(load=False)
>>> filename
'path/to/HeadMRVolume.mhd'
```
